contractsV5/KyberNetwork.sol

```diff
@@ -508,7 +508,8 @@ contract KyberNetwork is Withdrawable, Utils, IKyberNetwork, ReentrancyGuard {
         uint totalFeeWei;
         uint tradeWeiAmount;
```

Comment:
seems here in wei amount should also define if withFee
or 
noFee

Reply:
Ok. It's noFee.